internal/repos/github.go

```diff
@@ -389,6 +389,19 @@ func (s *GitHubSource) makeRepo(r *github.Repository) *types.Repo {
 	// so we don't want to store it.
 	metadata.ViewerPermission = ""
 	metadata.Description = sanitizeToUTF8(metadata.Description)
+
+	isPrivate := r.IsPrivate
+	if r.Visibility == "internal" {
+		switch s.config.InternalRepoVisibility {
+		case "public":
+			isPrivate = false
+		case "private":
+			isPrivate = true
+		default:
+			isPrivate = true
+		}
```

Comment:
I guess, can be simplified as:
```suggestion
		isPrivate = s.config.InternalRepoVisibility !== 'public'
```

Reply:
Be careful, your JavaScript is showing 😆 

But yeah definitely, thanks!